i18n/en.json

```diff
@@ -1,4 +1,9 @@
 {
+  "import": "Import",
+  "importType": "Import {{ type }}",
+  "importModalDescriptionStart": "Importing content requires a zip file. See our",
+  "importModalDescriptionLink": "guide to importing content",
+  "importModalDescriptionEnd": "in Apostrophe.",
```

Comment:
I do not like that

Reply:
but this is the only way I found to have an actual link in translations

Reply:
Possible to use `v-html` with `$t`?

Reply:
gg @falkodev 

![image](https://github.com/apostrophecms/import-export/assets/8301962/ef1dece5-298a-4ba7-b7d7-e0943e684b0c)

![image](https://github.com/apostrophecms/import-export/assets/8301962/a414858b-52a9-44d5-a5eb-a4fd8dfb17ab)

much better